guides/common/modules/proc_live-patching-hosts-using-kernelcare.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Live_Patching_Hosts_Using_KernelCare_Plug_In_{context}"]
+= Live Patching Hosts Using KernelCare Plug-In
+
+You can use KernelCare plug-in to live-patch the Linux kernel on managed hosts.
+
+.Prerequisites
+* Ensure your host has the `kernelcare` package installed.
+* Ensure your hosts runs {EL} 7, {EL} 8, or {EL} 9.
+* Ensure your host has access to the internet to connect to cloudlinux.com.
```

Comment:
I would stick either with singular or plural. Note that the title mentions multiple hosts.
```suggestion
* Ensure your hosts have the `kernelcare` package installed.
* Ensure your hosts run {EL} 7, {EL} 8, or {EL} 9.
* Ensure your hosts have access to the internet to connect to cloudlinux.com.
```

Reply:
Applied.